### DISCUSSION TITLE
[#991900] Z-fighting on selection animation texture

### DISCUSSION THREAD

**Discussion by MisledWater79** [Mojang/minecraft-editor#19] [Bugs]
It seems there is z-fighting and the animation seems to bug a bit on the selection around blocks.![PXL_20230308_190526416.jpg](https://user-images.githubusercontent.com/59891258/223812792-c06331fe-ebc8-4e8b-a9d2-b4c9d171a881.jpg)

I can't take screenshots well on my laptop so sorry for the bad quality picture but this shows the z-fighting on the right and the weird animation on the left (best picture since it goes by fast).

**Comment by minecraftchloe** [comment]
We have a fix for z-fighting that should be rolling out soon! Although, I'm not totally sure our fix will map to what you're seeing since that's a static picture -- ours is around Ctrl+Shift+V paste preview overlapping on a selection volume outline. In any case, be sure to let us know what you think.
